src/frontend/src/components/public/PIAFormTabs/additionalRisks/index.tsx

```diff
@@ -99,6 +128,17 @@ const PIAAdditionalRisks = () => {
 
   const columns = [{ name: 'Possible risk' }, { name: 'Response' }];
 
+  useEffect(() => {
+    if (!deepEqual(initialFormState, additionalRisksForm)) {
+      piaStateChangeHandler(additionalRisksForm, 'additionalRisks');
+    }
+  }, [
+    pia.additionalRisks,
```

Comment:
same for this dep `pia.additionalRisks`

You may result in a conflict to Bing's PR.